content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -634,8 +634,8 @@ The following operators can only be used with `nodeAffinity`.
 
 |    Operator    |    Behaviour    |
 | :------------: | :-------------: |
-| `Gt` | The field value will be parsed as an integer, and that integer is less than the integer that results from parsing the value of a label named by this selector |
-| `Lt` | The field value will be parsed as an integer, and that integer is greater than the integer that results from parsing the value of a label named by this selector |
+| `Lt` | The field value will be parsed as an integer, and that integer is less than the integer that results from parsing the value of a label named by this selector |
+| `Gt` | The field value will be parsed as an integer, and that integer is greater than the integer that results from parsing the value of a label named by this selector |
```

Comment:
thanks I got it ,but it will be better if you can explain that with a example below that context

Reply:
> The current content is correct, but the description text is confusing.

thanks I got it ,but it will be better if you can explain that with a example below that content

Reply:
Yes The description is confusing but the current content is right.  So Here We can Write 
| `Gt ` |  The field value will be parsed to an integer, and this integer must be greater than the integer value of the label specified by this selector.|
| `Lt ` |  The field value will be parsed to an integer, and this integer must be greater than the integer value of the label specified by this selector.|